internal/rukpak/source/containers_image.go

```diff
@@ -45,6 +57,17 @@ func (i *ContainersImageRegistry) Unpack(ctx context.Context, bundle *BundleSour
 	if err != nil {
 		return nil, err
 	}
+
+	//////////////////////////////////////////////////////
+	//
+	// Verify that registries configuration is up-to-date
+	//
+	//////////////////////////////////////////////////////
+	err = i.checkUpdatedConfiguration()
+	if err != nil {
+		return nil, err
```

Comment:
might be a good idea to wrap the error with some context, eg.   `fmt.Errorf("error checking registries configuration: %w", err)`, WDYT?